proprietary/tokens/src/components/ams/avatar.tokens.json

```diff
@@ -4,10 +4,10 @@
       "aspect-ratio": { "value": "{ams.aspect-ratio.square}" },
       "background-color": { "value": "{ams.color.highlight.purple}" },
       "color": { "value": "{ams.color.text.inverse}" },
-      "font-family": { "value": "{ams.text.font-family}" },
-      "font-size": { "value": "{ams.text.level.6.font-size}" },
-      "font-weight": { "value": "{ams.text.font-weight.normal}" },
-      "line-height": { "value": "{ams.text.level.6.line-height}" },
+      "font-family": { "value": "{ams.typography.font-family}" },
+      "font-size": { "value": "{ams.typography.body-text.small.font-size}" },
+      "font-weight": { "value": "{ams.typography.body-text.font-weight}" },
+      "line-height": { "value": "{ams.typography.body-text.small.line-height}" },
```

Comment:
You mentioned this @alimpens  – let’s double check.

This or `{ams.typography.line-height.body-text.small}`?